script.js

```diff
@@ -0,0 +1,65 @@
+function invokeRZP() {
+    var data = razorpay_script_vars.data;
+
+    var setDisabled = function(id, state) {
+        if (typeof state === "undefined") {
+            state = true;
+        }
+
+        var elem = document.getElementById(id);
+
+        if (state === false) {
+            elem.removeAttribute("disabled");
+        } else {
+            elem.setAttribute("disabled", state);
+        }
+    };
+
+    // Payment was closed without handler getting called
+    data.modal = {
+        ondismiss: function() {
+            setDisabled("btn-razorpay", false);
+        }
+    };
+
+    data.handler = function(payment) {
+        setDisabled("btn-razorpay-cancel");
+
+        var successMsg = document.getElementById("msg-razorpay-success");
+
+        successMsg.style.display = "block";
+
+        document.getElementById("razorpay_payment_id").value =
+            payment.razorpay_payment_id;
+
+        document.getElementById("razorpay_signature").value =
+            payment.razorpay_signature;
```

Comment:
@mayankamencherla I think this is just prettier enforcing the 80 character limit.